content/github/administering-a-repository/about-protected-branches.md

```diff
@@ -86,8 +86,8 @@ You can set up required status checks to either be "loose" or "strict." The type
 
```

Comment:
Thanks so much for opening an issue and PR for this fix @brian-armory! I can't comment on the line above ☝️ but there's one more "up-to-date" on line 85. Could you update that one, as well, please?

Reply:
Done!